js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadAllowedRoot.tsx

```diff
@@ -27,6 +27,7 @@ interface Props {
   pipelinePath: string;
   repoAddress: RepoAddress;
   sessionPresets?: Partial<IExecutionSession>;
+  onSaveConfig?: (config: any) => void;
```

Comment:
Ideally we can be stricter about the type here than `any`, which basically just sidesteps TypeScript altogether for this value (and any typechecking downstream of it).